build-tools/automation/azure-pipelines.yaml

```diff
@@ -493,6 +476,196 @@ stages:
         targetPath: $(Build.ArtifactStagingDirectory)
       condition: always()
 
+  # Check - "Xamarin.Android (Test MSBuild Mac)"
+  - job: mac_msbuild_tests
+    displayName: MSBuild Mac
+    pool: $(XA.Build.Mac.Pool)
+    timeoutInMinutes: 180
+    cancelTimeoutInMinutes: 5
+    workspace:
+      clean: all
+    steps:
+    - template: yaml-templates/mac/setup-test-environment.yaml
+
+    - task: DownloadPipelineArtifact@1
+      inputs:
+        artifactName: $(NUnitTestAssembliesArtifactName)
+        downloadPath: $(System.DefaultWorkingDirectory)/bin/Test$(XA.Build.Configuration)
+
+    - script: >
+        mono packages/NUnit.ConsoleRunner.3.9.0/tools/nunit3-console.exe
```

Comment:
Can we put the path to `nunit3-console.exe` in a variable?

We will probably update it to something newer than 3.9.0 at some point.